webpack/assets/javascripts/react_app/redux/API/APIRequest.js

```diff
@@ -13,7 +23,7 @@ export const get = async (
     payload: modifiedPayload,
   });
   try {
-    const { data } = await API.get(url, headers, params);
+    const { data } = await getApiResponse({ type, url, headers, params });
```

Comment:
Does it need to await here if there is an `await` in `getApiResponse`?

Reply:
thanks, this `await` should remain in here,
although it seem the the `await` in the helpers could be removed.